res/field/encounters_ex/great_marsh_dailies_natdex.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
**suggestion**: Combine the Great Marsh daily lookout encounters into a single file, including the coordinates that they map to, if that is known.